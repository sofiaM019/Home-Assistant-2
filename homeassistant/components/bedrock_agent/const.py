"""Constants for the Amazon Bedrock Agent integration."""

from typing import Final

DOMAIN: Final = "bedrock_agent"

CONST_KEY_ID: Final = "key_id"
CONST_KEY_SECRET: Final = "key_secret"
CONST_REGION: Final = "region"
CONST_MODEL_ID: Final = "model_id"
CONST_KNOWLEDGEBASE_ID: Final = "knowledgebase_id"
CONST_AGENT_ID: Final = "agent_id"
CONST_AGENT_ALIAS_ID: Final = "agent_alias_id"
CONST_TITLE: Final = "title"

CONST_PROMPT_CONTEXT: Final = "prompt_context"
CONST_MODEL_LIST: Final = [
    "amazon.titan-text-express-v1",
    "amazon.titan-text-lite-v1",
    "anthropic.claude-v2",
    "anthropic.claude-v2:1",
    "anthropic.claude-instant-v1",
    "ai21.j2-mid-v1",
    "ai21.j2-ultra-v1",
    "cohere.command-text-v14",
    "cohere.command-light-text-v14",
    "cohere.command-r-v1:0",
    "cohere.command-r-plus-v1:0",
    "meta.llama2-13b-chat-v1",
    "meta.llama2-70b-chat-v1",
    "meta.llama3-8b-instruct-v1:0",
    "meta.llama3-70b-instruct-v1:0",
    "mistral.mistral-7b-instruct-v0:2",
    "mistral.mixtral-8x7b-instruct-v0:1",
    "mistral.mistral-large-2402-v1:0",
    "mistral.mistral-small-2402-v1:0",
]

CONST_SERVICE_PARAM_PROMPT: Final = "prompt"
CONST_SERVICE_PARAM_MODEL_ID: Final = "model_id"
CONST_SERVICE_PARAM_IMAGE_URLS: Final = "image_urls"
CONST_SERVICE_PARAM_FILENAMES: Final = "image_filenames"
