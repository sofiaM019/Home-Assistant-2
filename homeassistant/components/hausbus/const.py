"""Constants for the Haus-Bus integration."""
import logging

LOGGER = logging.getLogger(__package__)

DOMAIN = "hausbus"
ATTR_ON_STATE = "on_state"
