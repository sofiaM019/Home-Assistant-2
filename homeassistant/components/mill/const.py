"""Constants for the Mill heater component."""

ATTR_AWAY_TEMP = "away_temp"
ATTR_COMFORT_TEMP = "comfort_temp"
ATTR_MAX_HEATING_POWER = "max_heating_power"
ATTR_ROOM_NAME = "room_name"
ATTR_SLEEP_TEMP = "sleep_temp"
BATTERY = "battery"
CLOUD = "Cloud"
CONNECTION_TYPE = "connection_type"
CONSUMPTION_TODAY = "day_consumption"
CONSUMPTION_YEAR = "year_consumption"
CONTROL_SIGNAL = "control_signal"
CURRENT_POWER = "current_power"
DOMAIN = "mill"
ECO2 = "eco2"
HUMIDITY = "humidity"
LOCAL = "Local"
MANUFACTURER = "Mill"
MAX_TEMP = 35
MIN_TEMP = 5
SERVICE_SET_ROOM_TEMP = "set_room_temperature"
SERVICE_MAX_HEATING_POWER = "max_heating_power"
TEMPERATURE = "current_temp"
TVOC = "tvoc"
