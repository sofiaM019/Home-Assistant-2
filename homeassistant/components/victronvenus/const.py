"""Constants for the victronvenus integration."""

CONF_INSTALLATION_ID = "installation_id"
CONF_MODEL = "model"
CONF_SERIAL = "serial"

DOMAIN = "victronvenus"

TOPIC_INSTALLATION_ID = "N/+/system/0/Serial"

DEVICE_MESSAGE = "device"
SENSOR_MESSAGE = "sensor"

PLACEHOLDER_PHASE = "{phase}"

DEFAULT_HOST = "venus.local."
DEFAULT_PORT = 1883
