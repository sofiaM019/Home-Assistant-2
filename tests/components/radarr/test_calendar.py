"""The tests for Radarr calendar platform."""

from datetime import timedelta

from freezegun.api import FrozenDateTimeFactory
import pytest
from syrupy import SnapshotAssertion

from homeassistant.const import STATE_OFF
from homeassistant.core import HomeAssistant
import homeassistant.util.dt as dt_util

from . import setup_integration

from tests.common import async_fire_time_changed
from tests.test_util.aiohttp import AiohttpClientMocker


@pytest.mark.parametrize(
    ("tested_time", "zone"),
    [
        (
            "2021-12-02 00:00:00-08:00",
            "US/Pacific",
        ),
        (
            "2021-12-03 09:00:00+01:00",
            "Europe/London",
        ),
    ],
    ids=("Pacific", "London"),
)
async def test_calendar(
    hass: HomeAssistant,
    aioclient_mock: AiohttpClientMocker,
    freezer: FrozenDateTimeFactory,
    snapshot: SnapshotAssertion,
    tested_time: str,
    zone: str,
) -> None:
    """Test for successfully setting up the Radarr platform."""
    tz = await dt_util.async_get_time_zone(zone)
    dt_util.set_default_time_zone(tz)
    freezer.move_to(tested_time)
    await setup_integration(hass, aioclient_mock)

    assert hass.states.get("calendar.mock_title") == snapshot

    freezer.tick(timedelta(days=1))
    async_fire_time_changed(hass)
    await hass.async_block_till_done()

    state = hass.states.get("calendar.mock_title")
    assert state.state == STATE_OFF
    assert len(state.attributes) == 1
    assert state.attributes.get("release_type") is None
