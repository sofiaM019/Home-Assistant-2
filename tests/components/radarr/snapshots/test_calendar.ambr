# serializer version: 1
# name: test_calendar[Pacific]
  StateSnapshot({
    'attributes': ReadOnlyDict({
      'all_day': True,
      'description': 'test2',
      'end_time': '2021-12-03 00:00:00',
      'friendly_name': 'Mock Title',
      'location': '',
      'message': 'test',
      'release_type': 'physicalRelease',
      'start_time': '2021-12-02 00:00:00',
    }),
    'context': <ANY>,
    'entity_id': 'calendar.mock_title',
    'last_changed': <ANY>,
    'last_reported': <ANY>,
    'last_updated': <ANY>,
    'state': 'on',
  })
# ---
# name: test_calendar[London]
  StateSnapshot({
    'attributes': ReadOnlyDict({
      'all_day': True,
      'description': 'test2',
      'end_time': '2021-12-04 00:00:00',
      'friendly_name': 'Mock Title',
      'location': '',
      'message': 'test',
      'release_type': 'physicalRelease',
      'start_time': '2021-12-03 00:00:00',
    }),
    'context': <ANY>,
    'entity_id': 'calendar.mock_title',
    'last_changed': <ANY>,
    'last_reported': <ANY>,
    'last_updated': <ANY>,
    'state': 'on',
  })
# ---
